exo-go/src/docker_compose/index.go

```diff
@@ -0,0 +1,43 @@
+package dockerCompose
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os/exec"
+)
+
+// RunAllImages runs all the docker images
+func RunAllImages(env []string, cwd string, log func(string)) (*exec.Cmd, *bytes.Buffer, error) {
+	return start(cwd, env, log, "docker-compose", "up")
+}
+
+// KillAllContainers kills all the containers
+func KillAllContainers(env []string, cwd string, log func(string)) (*exec.Cmd, *bytes.Buffer, error) {
+	return start(cwd, env, log, "docker-compose", "down")
+}
+
+func start(dir string, env []string, log func(string), command ...string) (*exec.Cmd, *bytes.Buffer, error) {
```

Comment:
We could make our own struct to wrap `exec.Command` with the useful functions we need. That way we can setup all the needed parameters before starting by calling methods on the structs instead of having to put it into a single function call. It could look something like this
```go
process = processHelpers.NewProcess("arg1", "arg2", "arg3")
process.SetDir(dir)
process.SetEnv(env)
process.SetStdoutLog(func(string) {})
process.Start()
process.GetCmd() // if you need the underlying cmd
```